### DISCUSSION TITLE
Reset polarity implementation (for At89S52/At89S51 etc..)

### DISCUSSION THREAD

**Discussion by afiglee** [avrdudes/avrdude#1825] [Q&A] [answered]
I cannot find anything related to inverted reset polarity. Even though Support to At89S51/At89S52 seem added.
Does it implies it has to be implemented in hardware on programmer's side or am I missing something?

**Comment by dl8dtl** [comment]
AFAICT this is only supported for the STK500 which indeed implements that in hardware.

**Reply by mcuee** [reply, marked as answer]
Or suitable usbasp FW which does that in FW. All in all, from the programmer side (either HW or FW).
* https://github.com/avrdudes/avrdude/issues/1304
* https://github.com/dioannidis/usbasp/releases/tag/v1.11
